### DISCUSSION TITLE
Multiple samples/vcf files

### DISCUSSION THREAD

**Discussion by FriederikeHanssen** [cccnrc/plot-VCF#3] [General]
IS it possible to plot multiple samples either all in one vcf file or load multiple vcf files at once?

**Comment by cccnrc** [comment]
Hi @FriederikeHanssen, thanks for reaching out! `plotVCF` is designed to handle and plot multiple sample in the same VCF file. As example, you can sub-select which samples you want to plot with [SAMPLE](https://github.com/cccnrc/plot-VCF#sample-focused-plot), differentiate each sample variants with [SHAPE](https://github.com/cccnrc/plot-VCF#sample-focused-plot---differentiate), group samples with [COLOR_SAMPLE](https://github.com/cccnrc/plot-VCF#sample-focused-plot---color-groups), and [much more](https://github.com/cccnrc/plot-VCF)!

It is not currently possible to load multiple VCF files into `plotVCF` (if you are interested in helping develop this option we are more than happy to help you out :wink:), but you can always merge your different VCF files first (take a look at [vcftools merge-vcf](https://cyverse.atlassian.net/wiki/spaces/DEapps/pages/259063974/vcftools-merge+0.1.16#:~:text=vcftools%2Dmerge%20merges%20two%20or,concatenating%20VCFs%20split%20by%20chromosome.) or [bcftools merge](https://samtools.github.io/bcftools/bcftools.html#merge)) and then pass the VCF file to `plotVCF`

I hope it was helpful, do not hesitate to further comment in case you need additional help

**Reply by FriederikeHanssen** [reply]
Ah great, thank you, missed this earlier. I am not sure my R skills are sufficient to contribute code :D .  But merging the vcf files beforehand has worked. My vcf files are quite big (think 80 WGS samples, 50X) , so I am running into memory issues, however this is not very special and I have expected this.

**Reply by cccnrc** [reply]
You can [divide your merged VCF per chromosome](https://www.biostars.org/p/9506417/) (or any other division criteria you like) and run `plotVCF` on each of them. It is a bit more of work, but it will work as soon as you get to an input VCF size which is compatible with your RAM capacity

**Reply by FriederikeHanssen** [reply]
yes, true. :) Thanks, for providing this awesome tool. Getting a quick overview is much easier
